doc/tutorials/image/image.txt

```diff
@@ -5,32 +5,32 @@
 
 @page imagetutorial Image Utilities tutorial
 
-@section intro_imagetut Introduction
+@section intro_imagetu Introduction
 
 Image datasets are becoming increasingly popular in deep learning.
 
 mlpack's image saving/loading functionality is based on [stb/](https://github.com/nothings/stb).
 
-@section toc_imagetut Table of Contents
+@section toc_imagetu Table of Contents
 
 This tutorial is split into the following sections:
 
- - \ref intro_imagetut
- - \ref toc_imagetut
- - \ref model_api_imagetut
- - \ref imageinfo_api_imagetut
- - \ref load_api_imagetut
- - \ref save_api_imagetut
+ - \ref intro_imagetu
+ - \ref toc_imagetu
+ - \ref model_api_imagetu
+ - \ref imageinfo_api_imagetu
+ - \ref load_api_imagetu
+ - \ref save_api_imagetu

```

Comment:
Probably not desired as such, but this did fix it. Revisit or accept?

Reply:
Accept!

Reply:
Sure, the names are just placeholders anyway.